lib/pg_ha_migrations/partman_config.rb

```diff
@@ -0,0 +1,7 @@
+class PgHaMigrations::PartmanConfig < ActiveRecord::Base
+  self.primary_key = :parent_table
+
+  def self.schema=(schema)
+    self.table_name = "#{schema}.part_config"
```

Comment:
Should we default this to `partman.part_config`?

Reply:
That doesn't seem necessary, as this class isn't meant to be used directly. In `safe_partman_update_config` we always discover the schema and call this method before interacting with the class

Reply:
I think we should add a comment here explaining what's going on. I think it makes sense now, but it's still pretty weird, and I'd like signposts in the future.

Reply:
Comments added